src/pymedphys/xlwings/_level1/dicom.py

```diff
@@ -81,3 +82,18 @@ def crossplane_profile(dicom_path, depth_adjust, depth_lookup, averaging_distanc
             ds, depth_adjust, depth_lookup, averaging_distance)
 
         return np.vstack([crossplane, crossplane_dose]).T
+
+    @xw.func
+    @xw.arg('dicom_path')
+    @xw.arg('depth_adjust')
+    @xw.arg('inplane')
+    @xw.arg('crossplane')
+    @xw.arg('depth')
+    @xw.ret(expand='table')
+    def arbitrary_profile(dicom_path, depth_adjust, inplane, crossplane, depth):
```

Comment:
Try to prepend the function name with a verb. =)

Reply:
What do you think about the following:

https://stackoverflow.com/a/1991341/3912576

Given there is no program state here being changed just values being returned given inputs is a verb required? I'm not sure?

Reply:
Further thought on this point, this function name ends up appearing within Excel. `arbitrary_profile` seems to be closer to the types of names found within Excel than say something like `extract_arbitrary_profile`.

Reply:
Happy to keep this as `arbitrary profile`. Prepending a verb is not a strong preference of mine, and the argument in the link you provided seems fair. It was more a feeling that most of our other code makes heavy use of words like `extract` (which is just a fancy `get`). Perhaps the answer is to adjust our other code, rather than add `extract` here.

Reply:
That makes sense